src/components/Author/Author.test.tsx

```diff
@@ -51,14 +41,13 @@ describe('<Author /> component', () => {
       latest: {
         name: 'verdaccio',
         version: '4.0.0',
+        dist: { fileCount: 0, unpackedSize: 0 },
       },
+      _uplinks: {},
```

Comment:
These are not needed for the test but were missing before. The type error was hidden by using `jest.fn`